app/networkapi/settings.py

```diff
@@ -45,7 +46,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG')
 
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
```

Comment:
I'm pretty sure `env` is the reason I can't run this app locally on windows (I can only run it in docker)
I can run the coss app no problem on windows though, so I feel like the `env` pattern might cater to UNIX like systems only.